internal/pkg/cli/dialog/inputs.go

```diff
@@ -85,7 +86,8 @@ func (f inputFields) Write(out *strings.Builder) {
 		if len(line.example) > 0 {
 			example = "<!-- " + line.example + " -->"
 		}
-		out.WriteString(strings.TrimSpace(fmt.Sprintf(format, line.mark, line.inputId, line.fieldPath, example)))
+		// Field path is escaped, it can contain MarkDown special chars, eg. _ []
+		out.WriteString(strings.TrimSpace(fmt.Sprintf(format, line.mark, line.inputId, "`"+line.fieldPath+"`", example)))
```

Comment:
Cesta ku hodnote moze obsahovat napr. index pola `foo.bar[123]` a Mardown to berie ako odkaz, preto hodnotu obalime do "`":
![image](https://user-images.githubusercontent.com/19371734/154272400-42f94436-fbc3-4e8a-932f-4542060cb6d9.png)